mimoto-mock-identity-wellknown.json

```diff
@@ -110,7 +110,7 @@
           "name": "Mock Verifiable Credential",
           "locale": "en",
           "logo": {
-            "url": "https://{mosip.api.public.host}/logo.png",
+            "url": "https://${mosip.api.public.host}/inji/logo.png",
```

Comment:
use mosip-logo.png as logo doesn't exist in this file server

Reply:
okay